src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/beregning/LæremidlerBeregnUtil.kt

```diff
@@ -24,7 +26,8 @@ object LæremidlerBeregnUtil {
                     val håndterNyUtbetalingsperiode = vedtaksperiode.håndterNyUtbetalingsperiode(acc)
                     acc + håndterNyUtbetalingsperiode
                 }
-            }.toList()
+            }.filter { it.vedtaksperioder.any { it.alleDatoer().any { !it.lørdagEllerSøndag() } } }
```

Comment:
Kan vurdere å trekke dette ut i en egen funksjon? 
For eksempel `vedtaksperioder.harDatoerIUkedager()`?

Reply:
Godt innspill! https://github.com/navikt/tilleggsstonader-sak/pull/559/commits/3451a95e24c27c7b6e7dd6fdbfc9d73af0734fd5